### PR TITLE
Added redux course in hindi and updated heading of ReactNative courses

### PR DESCRIPTION
**Pull request by hakiKhuva**
## What does this PR do?
Add resource(s)

### Description
Added Redux videos in Hindi

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -419,6 +420,13 @@
 * [React Native Tutorials for Begginers](https://youtube.com/playlist?list=PL8kfZyp--gEXs4YsSLtB3KqDtdOFHMjWZ) - Programming with Mash
 
 
+#### Redux
+
+* [Redux toolkit crash course | Chai aur React Series](https://youtu.be/1i04-A7kfFI?si=_0I8vhyhK_5ISJEg) - Chai aur Code
```
Comment (eshellman) [inline courses/free-courses-hi.md:425]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
no shortened links
remove trang (`si` ) parameters

Comment (hakiKhuva) [reply courses/free-courses-hi.md]: updated links

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -419,6 +420,12 @@
 * [React Native Tutorials for Begginers](https://youtube.com/playlist?list=PL8kfZyp--gEXs4YsSLtB3KqDtdOFHMjWZ) - Programming with Mash
 
 
+#### Redux
+
+* [Redux toolkit crash course | Chai aur React Series](https://www.youtube.com/watch?v=1i04-A7kfFI) - Hitesh Choudhary (Chai aur Code)
```
Comment (eshellman) [inline courses/free-courses-hi.md:425]: escape the `|`

Comment (hakiKhuva) [reply courses/free-courses-hi.md]: Done

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**